live-examples/css-examples/@counter-style/symbols.css

```
@@ -0,0 +1,8 @@
+@counter-style cool-symbols {
+    system: symbolic;
+    symbols: '*' '\21C9'▲ 🐉 '🥲' ✹ ❎ 💓;
```

```suggestion
    symbols: "*" "\21C9" ▲ 🐉 "🥲" ✹ '"' ❎ 💓;
```
i added a double quote here to show that quotes can be included